### PR TITLE
fix(ui5-label): rename wrap property to wrappingType

### PR DESCRIPTION
**Pull request by gmkv**
Part of #3107

BREAKING_CHANGE: property wrap has been renamed to wrappingType and its type has changed

### CODE REVIEW

File: packages/main/src/CheckBox.js [hunk 1]
```diff
@@ -290,7 +290,9 @@ class CheckBox extends UI5Element {
 	syncLabel() {
 		this._label = { ...this._label };
 		this._label.text = this.text;
-		this._label.wrap = this.wrap;
+		/* temporary workaround. remove after all wrap properties in the relevant components are renamed to wrappingType */
+		this._label.wrappingType = this.wrap ? "Normal" : "None";
```
Comment (gmkv) [inline packages/main/src/CheckBox.js:294]: Should we add a wrappingType prop to CheckBox?

File: packages/main/src/RadioButton.js [hunk 2]
```diff
@@ -258,6 +258,11 @@ class RadioButton extends UI5Element {
 
 	onBeforeRendering() {
 		this.syncGroup();
+
+		/* temporary workaround. remove after all wrap properties in the relevant components are renamed to wrappingType */
+		this._wrappingType = this.wrap ? "Normal" : "None";
```
Comment (gmkv) [inline packages/main/src/RadioButton.js:263]: Should we add a wrappingType prop to RadioButton?

File: packages/main/src/CheckBox.js (hunk 1 above)
Comment (ilhan007) [reply packages/main/src/CheckBox.js]: Hi @gkmv we should, but perhaps as part of a separate change

Comment (gmkv) [reply packages/main/src/CheckBox.js]: Ok. I will create a follow-up change.

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**